packages/chaire-lib-backend/src/utils/processManagers/__tests__/TrRoutingProcessManager.test.ts

```diff
@@ -31,9 +39,12 @@ startProcessMock.mockImplementation(async (serviceName: string,
     service: tagName,
     name: serviceName
 }));
+let defaultConfig = _cloneDeep(config);
```

Comment:
Maybe defaultTestConfig ?    
It kind of confuse me. 
Maybe a comment about why we need this cloneDeep. 
Should this be done in the beforeEach instead?

Reply:
Comment still valid.

Reply:
Added